os-cinder.te

```diff
@@ -17,11 +17,11 @@ gen_require(`
 allow httpd_t cinder_log_t:file { open create };
 allow httpd_t cinder_log_t:dir { add_name write };
 
-# Bugzilla 1820504
-allow httpd_t cinder_var_lib_t:dir { add_name write create };
+# Bugzilla 1820504 and 2053849
+allow httpd_t cinder_var_lib_t:dir { add_name write create remove_name };
```

Comment:
The supplied logs seem to also involve denials for "read". I think at this point we can probably use the manage_dirs_pattern macro directly:

`manage_dirs_pattern(httpd_t, cinder_var_lib_t, cinder_var_lib_t);`